src/config/ConfigManager.cpp

```diff
@@ -1566,7 +1567,11 @@ void CConfigManager::loadConfigLoadVars() {
         g_pHyprOpenGL->m_bReloadScreenShader = true;
 
     // parseError will be displayed next frame
+#ifdef HYPRLAND_DEBUG
+    if (parseError != "" && !configValues["debug:suppress_errors"].intValue)
```

Comment:
no reason to guard this for only debug. `debug:` options are generally available.

Reply:
My reasoning is that if some users used this option in the hyprland release build and forgot about it. then reported a issue when they found their configuration did not work...

Well, that would be very time-consuming.

Reply:
So, should I remove the preprocessor guard?

Reply:
yes

Reply:
Removed the preprocessor guard, thus marked as resolved.